Update Italian.html

AbdallahQashmar opened the pull request:
still need to edit the links and the picture sizes.

ysmeen24 committed:
Update WelcomePage.html

I updated the welcome page please review.

ysmeen24 committed:
Merge pull request #30 from ysmeen24/WelcomePage-1

Update WelcomePage.html

ysmeen24 committed:
Create README.md

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

ysmeen24 committed:
Update WelcomePage.html

AbdallahQashmar committed:
Update Italian.html

still need to edit the links and the picture sizes.

ysmeen24 reviewed (changes requested):
Everything looks great Abdullah. But as you said please check the links for the images and the dimensions. Thank you

AbdallahQashmar committed:
Update Italian.html

Everything is done with the italian cuisine, notify me if there's anything wrong.
